tests/NRedisStack.Tests/Search/SearchTests.cs

```diff
@@ -2752,4 +2752,23 @@ public void TestProfileCommandBuilder()
         Assert.Equal(new object[] { "index", "SEARCH", "LIMITED", "QUERY", "*" }, search.Args);
         Assert.Equal(new object[] { "index", "AGGREGATE", "LIMITED", "QUERY", "*" }, aggregate.Args);
     }
+
+    [Fact]
+    public void Issue175()
+    {
+        ConnectionMultiplexer redis = ConnectionMultiplexer.Connect("localhost:6379");
+        IDatabase db = redis.GetDatabase();
+        db.Execute("FLUSHALL");
+
+        SearchCommands ft = db.FT();
+
+        var sortable = true;
+        //Add a tag field with sortable = true
+        var schema = new Schema().AddTagField("", sortable, false, false, "|");
+
+        //following line throws exception (Note: sortable = true)
+        Assert.True(ft.Create("myIndex", new FTCreateParams()
+                .On(IndexDataType.JSON)
+                .Prefix("doc:"), schema));
```

Comment:
Comment might be a bit misleading considering the assertion tests true.

Minor thing, but maybe split the param creation and FT.CREATE into variables for readability?